### PR TITLE
Added homework_21_140223

### PR DESCRIPTION
**Pull request by kate-lvlup**
Hi Dima,
Could you check my homework21?

Thanks in advance!

### CODE REVIEW

File: homework_21_140223/src/AccountUtils.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        return accountList.stream()
+                .filter(account -> account.getCountry().equals("Brazil"))
+                .map(account -> account.getFirstName() + " + " + account.getLastName())
+                .collect(Collectors.toList());
+    }
+
+    //Method3
+    public static Optional<Account> returnOptionalAccountWithLargestBalance(List<Account> accountList) {
+        return accountList.stream()
+                .max((o1, o2) -> {
```
Comment (YeaaaaH) [inline homework_21_140223/src/AccountUtils.java:46]: simplify to Integer::compareTo

File: homework_21_140223/src/AccountUtils.java [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                    } else {
+                        return 0;
+                    }
+                });
+    }
+
+    //Method4
+    public static void returnAllUsersWithSpecMonthOfBirth(List<Account> accountList, int monthInt) {
+        accountList = accountList.stream()
+                .filter(new Predicate<Account>() {
```
Comment (YeaaaaH) [inline homework_21_140223/src/AccountUtils.java:60]: simplify to lambda

Comment (kate-lvlup) [reply homework_21_140223/src/AccountUtils.java:60]: Fixed

File: homework_21_140223/src/AccountUtils.java [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+            System.out.println();
+        }
+//        Another solution by teacher!!!
+//        newGroupMap.entrySet()
+//                .forEach(x -> System.out.println(x));
+    }
+
+    //Method9
+    public static long returnNumberOfAccountsWithAgeAbove30Years(List<List<Account>> listOfListsOfAccounts) {
+        return listOfListsOfAccounts.stream()
```
Comment (YeaaaaH) [inline homework_21_140223/src/AccountUtils.java:111]: also can be simplified

**Review by YeaaaaH — CHANGES_REQUESTED** [review]
need to fix

File: homework_21_140223/src/AccountUtils.java (hunk 3 above)
Comment (kate-lvlup) [reply homework_21_140223/src/AccountUtils.java:111]: Fixed

File: homework_21_140223/src/AccountUtils.java (hunk 1 above)
Comment (kate-lvlup) [reply homework_21_140223/src/AccountUtils.java:46]: Fixed

**Review by YeaaaaH — APPROVED** [review]

**Pull request merged**